src/util/native_fs_utils.js

```diff
@@ -548,6 +548,9 @@ async function is_dir_rw_accessible(fs_context, dir_path) {
     } catch (err) {
         return false;
     }
+
+    if (config.NC_DISABLE_POSIX_MODE_ACCESS_CHECK) return true;
```

Comment:
Maybe you need to edit the function description with this change? (currently it is: "is_dir_rw_accessible validate the dir param accessible for read and write")